idsvr/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: idsvr
 version: 0.9.9
```

Comment:
I usually bump the chart version as well wen updating the appversion, not sure it is the best practice though